lib/ansible/modules/system/lvg.py

```diff
@@ -254,6 +262,31 @@ def main():
         devs_to_remove = list(set(current_devs) - set(dev_list))
         devs_to_add = list(set(dev_list) - set(current_devs))
 
+        if current_devs:
+            if state == 'present' and pvresize:
+                for device in current_devs:
+                    pvdisplay_cmd = module.get_bin_path('pvdisplay', True)
+                    pvdisplay_ops = "--units b --columns --noheadings --nosuffix"
+                    pvdiplay_cmd_device_options = pvdisplay_cmd + " " + device + " " + pvdisplay_ops
+                    rc, dev_size, err = module.run_command(pvdiplay_cmd_device_options + " -o dev_size")
```

Comment:
```suggestion
                    rc, dev_size, err = module.run_command(pvdiplay_cmd_device_options + ["-o", "dev_size"])
```

Reply:
Do you need to check `rc`? If you want to simply check whether `rc == 0` (and call `module.fail_json` otherwise), add `check_rc=True` to the `module.run_command` call.